web-app/src/services/holidayService.js

```diff
@@ -24,8 +24,16 @@ export const approveHoliday = eventId => {
   return axios.put('/holidays/approveHoliday', { eventId });
 };
 
-export const rejectHoliday = eventId => {
-  return axios.put('/holidays/rejectHoliday', { eventId });
+export const rejectHoliday = holidayId => {
+  return axios.put('/holidays/cancelHoliday', { holidayId });
+};
```

Comment:
Reject holiday is using the cancel endpoint?

Reply:
Curious if anything on this one? What are we editing? When you hit the cancelHoliday endpoint is it updating a particular field? I'd be careful with naming here. Cancel holiday would fall under what the user would be doing in my opinion. Reject holiday would be under an admin.

Reply:
The cancelHoliday endpoint is indeed triggered by the user’s action whereas reject is by the admin. I’m really confused why this was changed.